.github/workflows/changelogs/changelogs.py

```diff
@@ -350,9 +360,15 @@
         # Test capitalization
         if re.match(self.regex.WRONG_CAP_START, entry.entry) or re.search(self.regex.WRONG_CAP_AFTER, entry.entry):
             issues.append(Issue(IssueType.WRONG_CAP, entry.file, entry.line, entry.end_line))
-        # Test spacing
-        if re.search(self.regex.WRONG_SPACING, entry.entry):
-            issues.append(Issue(IssueType.WRONG_SPACING, entry.file, entry.line, entry.end_line))
+
+        # Test spacing (ignored in version strings)
+        # Test to check if a match overlaps with a version string
+        overlaps = lambda ver_str, match: ver_str.start() <= match.start() and ver_str.end() >= match.end()
+        for match in re.finditer(self.regex.WRONG_SPACING, entry.entry):
+            # Ignore if part of a version string
+            if not any(overlaps(ver_str, match) for ver_str in re.finditer(self.regex.VERSION_REGEX, entry.entry[:match.end()])):
```

Comment:
[nitpick] Our Python linter does not run on this file, if it did it would complain about assigning a lambda to a variable.

Reply:
What would be an accepted alternative here?

Reply:
```
def overlaps(ver_str, match):
    ...
```